app/services/google_credentials_generator.rb

```diff
@@ -67,7 +66,7 @@ def substitute_env_vars
     @client_secret = data['client_secret']
     data['client_secret'] = nil
 
-    fp = File.open('/Users/stephen/src/gds/inventory-tool/config/new-config.json', 'w')
+    fp = File.open('config/google_drive_config.json', 'w')
```

Comment:
hate to be picky but can we change `fp` to something more descriptive - `config_file` maybe?